### PR TITLE
Add question restrictions to polls

### PR DESCRIPTION
**Pull request by zoek1**
##### Description
- [X] minimum character count for open questions
- [X]  multiple choice (mandatory or not toggle)
- [X]  single choice (mandatory or not toggle)
- [X]  single option (leave as is)

https://www.loom.com/share/3ccdcc43a69d42c387481be328dddb29

##### Refers/Fixes
closes https://github.com/gitcoinco/web/issues/7181

### CODE REVIEW

File: app/dashboard/models.py [hunk 1]
```diff
@@ -5191,6 +5191,10 @@ class Question(SuperModel):
         ('OPEN', 'Open'),
     )
 
+    TYPE_RESTRICTIONS = (
```
Comment (thelostone-mc) [inline app/dashboard/models.py:5194]: Is this used anywhere ?

**Comment by frankchen07** [comment]
minimum character count for open questions looks good
 multiple choice (mandatory or not toggle) looks good
 single choice (mandatory or not toggle) looks good
 single option (leave as is) looks good

Can you show the functionality of where we can toggle each question on or off for the restriction?

otherwise LGTM

**Comment by zoek1** [comment]
@frankchen07 https://www.loom.com/share/71cb43bd4f51464da1d88570fdff0072

File: app/dashboard/models.py (hunk 1 above)
Comment (zoek1) [reply app/dashboard/models.py]: removed, thanks @thelostone-mc :)

**Comment by PixelantDesign** [comment]
yayyyy

**Pull request merged**